src/version.ts

```diff
@@ -0,0 +1 @@
+export const SERVER_VERSION = "2.2.0";
```

Comment:
we need to delete this file, it supposes that it's not versioned and its generated automatically?

Reply:
done